source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -61,24 +61,24 @@ content: |
   the ``medicalRecords.patients`` collection namespace:
 
   .. tabs-drivers::
-   
+
      .. tab::
         :tabid: java-sync
-        
+
         .. code-block:: java
            :emphasize-lines: 2
-           
+
            HashMap<String, BsonDocument> schemaMap = new HashMap<String, BsonDocument>() {{
              put("medicalRecords.patients", BsonDocument.parse(jsonSchema));
            }}
      .. tab::
         :tabid: nodejs
-        
+
         .. code-block:: javascript
-        
-           const dataNamespace = "medicalRecords.patients";
+           :emphasize-lines: 2
+
            const patientSchema = {
-             [dataNamespace]: jsonSchema
+             ["medicalRecords.patients"]: jsonSchema,
```

Comment:
If we're removing the use of the variable 'dataNamespace' there's no need for the array brackets around  "medicalRecords.patients". The array brackets around a keyname are only useful if you want the keyname comes from a bracket. Please change this to:

   ```
        const patientSchema = {
"medicalRecords.patients": jsonSchema,
}
```

Reply:
Will fix